index.bs

```diff
@@ -678,34 +880,87 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		<h4 method for="Clipboard">write(|data|)</h4>
 		The {{Clipboard/write(data)}} method must run these steps:
 
-			1. Let |p| be a new [=Promise=].
+			1. Let |realm| be [=this=]'s [=relevant realm=].
+
+			1. Let |p| be [=a new promise=] in |realm|.
 
 			1. Run the following steps [=in parallel=]:
 
-				1. Let |r| be the result of running [=check clipboard write permission=] [=in parallel=]
+				1. Let |r| be the result of running [=check clipboard write permission=].
+
+				Note: Clipboard permission is not supported on Safari. However, the write() method must be called inside
```

Comment:
This needs extra indents so that it nests under step 3.1. Otherwise the list restarts with a second 3.1 on the next line.

Reply:
Ack.

Reply:
Done.